lib/vhdl-linter.ts

```diff
@@ -169,7 +170,9 @@ export class VhdlLinter {
     let i = 0;
     start = Date.now();
     try {
-      await Elaborate.elaborate(this);
+      if (this.elaborated === false) {
+        await Elaborate.elaborate(this);
```

Comment:
I would prefer to set the `elaborated` flag in here instead of in the elaborate function